python/dolfinx/io/utils.py

```diff
@@ -18,8 +18,8 @@
 import basix.ufl
 import ufl
 from dolfinx import cpp as _cpp
-from dolfinx.cpp.io import perm_gmsh as cell_perm_gmsh  # noqa F401
-from dolfinx.cpp.io import perm_vtk as cell_perm_vtk  # noqa F401
+from dolfinx.cpp.io import perm_gmsh as cell_perm_gmsh  # F401
```

Comment:
Removing the `noqa` is odd: I guess it should have been either `noqa: F401`, or no comment at all